backend/src/services/ecoinfo.service.ts

```diff
@@ -10,7 +10,7 @@ import { z } from 'zod'
  *            información de paginación
  */
 export const getAllEcoinfo = async (): Promise<Ecoinfo[]> => {
-  // await fetchEcoInfo()
```

Comment:
Please leave this line commented, only change it on your local machine when testing explicitly ecoInfo, or else we will consume all the free tier requests